test/src/comps/test_util.cc

```diff
@@ -1,18 +1,17 @@
+#include "RandLAPACK.hh"
+#include "RandBLAS.hh"
```

Comment:
RandBLAS should probably be included with brackets.

Reply:
done